testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -627,9 +627,9 @@ public void testCommentColumnView()
     public void testCommentColumnMaterializedView()
     {
         String viewName = "comment_materialized_view" + randomNameSuffix();
-        assertUpdate("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM orders");
-        assertAccessDenied("COMMENT ON COLUMN " + viewName + ".orderkey IS 'new order key comment'", "Cannot comment column to .*", privilege(viewName, COMMENT_COLUMN));
-        assertUpdate(getSession(), "COMMENT ON COLUMN " + viewName + ".orderkey IS 'new comment'");
+        assertUpdate("CREATE MATERIALIZED VIEW mock.default." + viewName + " AS SELECT * FROM orders");
```

Comment:
Can we specify the operation type on commit message - commit title looks a bit generic

Reply:
refactored

Reply:
Can we rename the commit message as `mock connector` instead of `mock catalog`.